service/deployer/installer/helm/helm.go

```diff
@@ -219,14 +219,15 @@ func (i *HelmInstaller) Install(event eventerspec.DeploymentEvent) error {
 		return microerror.MaskAny(err)
 	}
 
-	if err := i.runHelmCommand(
-		"install",
-		"install",
-		tarballPath,
+	installCommand := []string{
+		"upgrade",
```

Comment:
Beforehand, upgrading was not working - we essentially were trying to install something that was already installed.